commit fa1d58c2df2f7c2e3b056e14d05048e7d58ae152
Author: Thorsten Leemhuis <linux@leemhuis.info>
Date:   Tue Apr 18 11:14:28 2017 -0700

    Input: elantech - add Fujitsu Lifebook E547 to force crc_enabled
    
    commit 704de489e0e3640a2ee2d0daf173e9f7375582ba upstream.
    
    Temporary got a Lifebook E547 into my hands and noticed the touchpad
    only works after running:
    
            echo "1" > /sys/devices/platform/i8042/serio2/crc_enabled
    
    Add it to the list of machines that need this workaround.
    
    Signed-off-by: Thorsten Leemhuis <linux@leemhuis.info>
    Reviewed-by: Ulrik De Bie <ulrik.debie-os@e2big.org>
    Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/drivers/input/mouse/elantech.c
===================================================================
--- linuxtest.orig/drivers/input/mouse/elantech.c
+++ linuxtest/drivers/input/mouse/elantech.c
@@ -1541,6 +1541,13 @@ static const struct dmi_system_id elante
 		},
 	},
 	{
+		/* Fujitsu LIFEBOOK E547 does not work with crc_enabled == 0 */
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "FUJITSU"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "LIFEBOOK E547"),
+		},
+	},
+	{
 		/* Fujitsu LIFEBOOK E554  does not work with crc_enabled == 0 */
 		.matches = {
 			DMI_MATCH(DMI_SYS_VENDOR, "FUJITSU"),
Index: linuxtest/dummy/rpi_1503_fa1d58c2df2f7c2e3b056e14d05048e7d58ae152.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1503_fa1d58c2df2f7c2e3b056e14d05048e7d58ae152.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
